### ISSUE TITLE
Data not available

### ISSUE THREAD

**Issue by ddurden** [khufkens/amerifluxr#8]
There seems to be some issues grabbing data from Ameriflux. I imagine it has to do with the move from ORNL to LBNL. This issue is seen in both download.ameriflux function and the Shiny app.

```{r} 
> dat <- amerifluxr::download.ameriflux(site = "US-Ho1")
Error in function (type, msg, asError = TRUE)  : 
  Server denied you to change to the given directory
Error in amerifluxr::download.ameriflux(site = "US-Ho1") : 
  last site, exiting
In addition: Warning message:
In amerifluxr::download.ameriflux(site = "US-Ho1") :
  site US-Ho1 does not exist, or system timeout!
```

Just wanted to let you know. I have acquired web services {json} links to grab metadata from AmeriFlux, but I haven't worked on data yet. If you are interested, I'm willing to help work on some fixes.

Cheers,
Dave D.

**Comment by khufkens** [comment]
You are correct, this has to do with the move and the fact they put everything behind a login wall making things like my application a pain to code up. I talked to Deb Agarwal about this back in October, but haven't heard back from here since. I'll contact her again. In the mean time if you manage to fix it I would be forever grateful as I'm strapped for time.

I fear however that implementing the login wall stuff will be a real pain. NASA has pulled a similar trick, inconveniencing the free flow of data in order to appease the bean counters.

**Comment by khufkens** [comment]
Oh, and could you provide me with the json hooks?

**Comment by ddurden** [comment]
Hey @khufkens,

Here are the json hooks that I was provided so far:

http://ameriflux-data.lbl.gov/AmeriFlux/SiteSearch.svc/PublishYears/AmeriFlux - returns a list of sites and the years that data is available for the site.
http://ameriflux-data.lbl.gov/AmeriFlux/SiteSearch.svc/SiteMapData/AmeriFlux - returns a list of sites, their location and some basic metadata.
http://ameriflux-data.lbl.gov/AmeriFlux/SiteSearch.svc/SiteList/AmeriFlux - returns a list with just the site id and site name.
http://ameriflux-data.lbl.gov/BADM/Anc/SiteInfo/<SITE_ID> - returns the site’s BADM information
http://ameriflux-data.lbl.gov/BADM/DOI/AmeriFlux/<SITE_ID> - returns the site’s DOI information

I inquired with Deb about data access as well. I will let you know what I hear.

Cheers,
DD

**Comment by khufkens** [comment]
Thanks, 

@ankurdesai also helped out with some code snippets from PECAN package. In between both these hints I should get this on the road again once I have some time.

I'll keep you posted.

K
